blockchain/storagev2/leveldb/leveldb.go

```diff
@@ -0,0 +1,100 @@
+package leveldb
+
+import (
+	"github.com/0xPolygon/polygon-edge/blockchain/storagev2"
+	"github.com/hashicorp/go-hclog"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+)
+
+// levelDB is the leveldb implementation of the kv storage
+type levelDB struct {
+	db *leveldb.DB
+}
+
+var tableMapper = map[uint8][]byte{
+	// Main DB
```

Comment:
Why do we use suffixes instead of prefixes?
Why lookup DB do not have suffixes at all?

Reply:
Why do we use suffixes instead of prefixes?
With suffixes write into main DB is sequential, with prefixes is random.

Why lookup DB do not have suffixes at all?
Because there is no need for suffixes, data is already unique.